core/object/object.cpp

```diff
@@ -939,7 +939,10 @@ void Object::set_meta(const StringName &p_name, const Variant &p_value) {
 		if (metadata.has(p_name)) {
 			metadata.erase(p_name);
 			metadata_properties.erase("metadata/" + p_name.operator String());
-			notify_property_list_changed();
+			if (p_name.operator String().find_char('_') != 0) // don't notify for editor-only properties
+			{
```

Comment:
```suggestion
			if (!p_name.operator String().begins_with('_')) { // Don't notify for editor-only properties.
```
Same below.

Reply:
Of course, I see your point. I pushed an edit that's basically the same as your suggestion, but with `"_"` instead as there doesn't seem to be a char version of begins_with

Reply:
You didn't fix the brace style though. `{` should come on the same line, as we do everywhere.

> with "_" instead

I just didn't notice you used a char.